pkg/controller/clone-controller.go

```diff
@@ -230,6 +231,23 @@ func (cc *CloneController) syncPvc(key string) error {
 	if !checkPVC(pvc, AnnCloneRequest) {
 		return nil
 	}
+
+	pvcPhase := pvc.Status.Phase
+	glog.V(3).Infof("PVC phase for PVC \"%s/%s\" is ", pvc.Namespace, pvc.Name, pvcPhase)
```

Comment:
I think you are missing a %s in the log message.

Reply:
Thanks.